frameworks/kafka/tests/test_sanity.py

```diff
@@ -85,6 +86,9 @@ def test_endpoints_zookeeper_default():
     )
 
 
+@pytest.mark.skip(
```

Comment:
🤔 Should we remove the test instead?

Reply:
We can remove this test but I kept it as a reference for the kafka versions<2.4.0. As this was an important test, I decided to skip it instead of removing it completely. WDYS?

Reply:
I'd remove it and provide a comment in the commit message instead.